packages/contracts/contracts/optimistic-ethereum/OVM/bridge/messaging/OVM_L2CrossDomainMessenger.sol

```diff
@@ -182,16 +182,30 @@ contract OVM_L2CrossDomainMessenger is
      * @return _valid Whether or not the message is valid.
      */
     function _verifyXDomainMessage()
-        view
         internal
         returns (
             bool _valid
         )
     {
+
+        address L1MessageSender = iOVM_L1MessageSender(resolve("OVM_L1MessageSender")).getL1MessageSender();
```

Comment:
`Lib_ExecutionManagerWrapper` will supersede `OVM_L1MessageSender` very soon. I would prefer to use it now if possible: https://github.com/ethereum-optimism/optimism/blob/8fee7bed6b63a861fa3e0300869d544bc3a75b45/packages/contracts/contracts/optimistic-ethereum/libraries/wrappers/Lib_ExecutionManagerWrapper.sol#L104-L121

Reply:
Also `L1MessageSender` should not be capitalized.